packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -3,16 +3,55 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
 import Input from 'terra-form/lib/Input';
+import ButtonGroup from 'terra-button-group';
+
 import TimeUtil from './TimeUtil';
 import styles from './TimeInput.scss';
 
 const cx = classNames.bind(styles);
 
+const validDateRegex = new RegExp('^(0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$');
+
+const FORMAT_12_HOUR = '12';
+const FORMAT_24_HOUR = '24';
+
+const inputType = {
+  HOUR: 0,
+  MINUTE: 1,
+};
+
+const keyCodes = {
```

Comment:
Moved to the TimeUtils file. This would align with [menu](https://github.com/cerner/terra-core/blob/master/packages/terra-menu/src/_MenuUtils.js#L18), [table](https://github.com/cerner/terra-core/blob/b2fd3f8eab22d0f0daca643226659a55d301b94f/packages/terra-table/src/SelectableUtils.js#L26), menu etc

Reply:
Done. https://github.com/cerner/terra-core/pull/1027/commits/8fa1c39d46b47d4e9ef34702f0fa8d50220b8e2d